src/components/tooltip/tooltip.js

```diff
@@ -175,8 +175,9 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $mdUtil, $mdThe
     }
 
     function showTooltip() {
-      // Insert the element before positioning it, so we can get the position
+      // Insert the element and position at top left, so we can get the position
       // and check if we should display it
+      element.css({top: 0, left: 0});
```

Comment:
It fixes a bug where the tooltip being displayed at its default location causes the scrollbar to appear and then the positioning of the tooltip is off when the tooltip moves and the scrollbar disappears. The bug: https://github.com/angular/material/issues/4345 

element.css({top:-100, left:0}) also works. I don't have a preference since either way works just as well.